simpleflu/README.md

```diff
@@ -105,22 +92,48 @@ This state prompts the meta agent to infect ten random agents and then wait inde
     }
 ```
 
+
 ## Sample Model Outputs
 
-This model can be run using the `METHODS` file, which is a `bash` script that runs the models and then uses `fred_plot` to generate a histogram of new infections per day (and week, not shown).
-This is consistent with the condition propagating through the specified population and creating enough `Recovered` individuals that the condition can no longer transmit.
+This model can be run using the **`METHODS`** file, which is a `bash` script that runs the model and uses `fred_plot` to generate a histogram of the number of new infections per day (and week, not shown).
+
+```bash
+$ ./METHODS
```

Comment:
I added a code block showing how to run the METHODS file and the standard output.